qbitunregistered.py

```
@@ -239,18 +255,36 @@
-        orphaned_files = check_files_on_disk(client, torrents, exclude_file_patterns=exclude_files, exclude_dirs=exclude_dirs)
-        logging.info("Total orphaned files: %d", len(orphaned_files))
-
-        # Delete orphaned files unless dry-run is set (pass torrents to avoid redundant API call)
-        delete_orphaned_files(orphaned_files, dry_run, client, torrents=torrents)
-        operation_results["succeeded"].append("Orphaned file check")
+        # Avoid treating recycle bin contents as orphaned on subsequent runs
+        recycle_bin = config.get("recycle_bin")
+        exclude_dirs_for_scan = list(exclude_dirs)
+        if recycle_bin:
+            # Convert to absolute path to ensure proper exclusion
+            exclude_dirs_for_scan.append(str(Path(recycle_bin).resolve()))
```

Ruff (0.14.5)

Undefined name `Path`

(F821)